### PR TITLE
chore(unity): refine endpoints to only return whats needed

### PR DESCRIPTION
**Pull request by ischolten**
Worked with @subirjolly and @gspletzer to update unity swagger so that the api is returning what the UI needs. 

- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [ ] Rebased/mergeable
- [ ] Tests pass

### CODE REVIEW

File: src/client/swagger/unity.yml
Comment (marcdel) [inline src/client/swagger/unity.yml:138]:
```suggestion
  '/settings/notifications':
```

**Review by randycoulman — APPROVED** [review]
I didn't deep dive on this, but from what I can see, it looks reasonable to me as a starting point. I expect we'll learn some things as we implement it and will make some tweaks.

Approving now so we can get this in before migrating it to a new home, but please make @marcdel's suggested spelling change before merging.

File: src/client/swagger/unity.yml
Comment (marcdel) [inline src/client/swagger/unity.yml:15]: Maybe I'm thinking of this too much as a traditional REST API, but it feels really strange everything has this implicit coupling to the user's session. Though I suppose the other option is `/account/:id` and then you're implicitly coupled to the session by way or authorization (checking the logged in user's account id matches the requested one) anyway.

File: src/client/swagger/unity.yml
Comment (marcdel) [inline src/client/swagger/unity.yml:591]: I can see this one being tricky depending on the context(s) in which it's used

File: src/client/swagger/unity.yml
Comment (marcdel) [inline src/client/swagger/unity.yml:26]: I don't love this top level resource being _a_ billing, because that's...not a noun. I'm not a fan of slapping an `info` suffix on things either though, and I can't think of a better name for this data clump.

These fields make sense under `Account` i.e. account.balance, account.payment_method, etc. but then you've got drag all these extra fields around anywhere you want an account.

File: src/client/swagger/unity.yml
Comment (marcdel) [inline src/client/swagger/unity.yml:62]: Is this something that could be returned as part of the `/billing` payload rather than making a separate call for it?

File: src/client/swagger/unity.yml
Comment (marcdel) [inline src/client/swagger/unity.yml:99]: I feel weird about having these at the same level as `/usage/vectors`. Assuming `{type}`s are vectors, I would expect this to be `/usage/vectors/:type`. If that's the case I'd probably make the route `/usage_vectors` instead?

File: src/client/swagger/unity.yml
Comment (marcdel) [inline src/client/swagger/unity.yml:111]: Are type and vector synonyms? If so, we should prob pick one or the other for these two endpoints.

File: src/client/swagger/unity.yml
Comment (marcdel) [reply src/client/swagger/unity.yml:138]: Is this distinct enough from a `Billing` to warrant its own object/route? Wonder if it makes sense to roll this under that section.

File: src/client/swagger/unity.yml
Comment (marcdel) [reply src/client/swagger/unity.yml:15]: Don't 🔫 me but if I were starting from scratch I'd probably lean more towards a use-case based model than a resource based one. Then split out resources if/when duplication becomes apparent. That's personal preference though, and I think the scope of these pages is small enough (for now) that either way will work fine.

File: src/client/swagger/unity.yml
Comment (gspletzer) [reply src/client/swagger/unity.yml:26]: That's a good point, but also can't think of a better name right now; will consider other options as we implement it.

File: src/client/swagger/unity.yml
Comment (gspletzer) [reply src/client/swagger/unity.yml:62]: That's a good point. This isn't really for billing; it's part of the usage page. We will see if we can include it in the usage vectors payload or similar.

File: src/client/swagger/unity.yml
Comment (gspletzer) [reply src/client/swagger/unity.yml:111]: Will replace `type` with `vector_name`

File: src/client/swagger/unity.yml
Comment (gspletzer) [reply src/client/swagger/unity.yml:138]: Because there is a `put` route for it as well, it makes sense to keep it separated.

File: src/client/swagger/unity.yml
Comment (gspletzer) [inline src/client/swagger/unity.yml:681]: This description should probably specify Quartz ID
```suggestion
        id:
          description: the quartz id of the invite
          readOnly: true
          type: integer
```

File: src/client/swagger/unity.yml
Comment (OfTheDelmer) [inline src/client/swagger/unity.yml:341]: It might be worth noting in the response that you will get a `404` from the delete on a retry (not idempotent) or if the org id doesn't match. This Quartz behavior differs from IDPE behavior where @glinton has structured deletes to be idempotent.

Comment (OfTheDelmer) [reply src/client/swagger/unity.yml:341]: @randycoulman thoughts?

Comment (randycoulman) [reply src/client/swagger/unity.yml:341]: My initial thought is that a 404 is the right response if the org ID is missing. But if there is no invite with that ID, then we should return a successful response in order to make this idempotent.

File: src/client/swagger/unity.yml
Comment (randycoulman) [reply src/client/swagger/unity.yml:681]: Invites only exist in Quartz and not IDPE, so it's pretty much implied that this is a Quartz ID. But not a bad idea to be explicit about it.

File: src/client/swagger/unity.yml
Comment (OfTheDelmer) [reply src/client/swagger/unity.yml:341]: Agreed, I'll update our issue to reflect retry should return a successful response.

File: src/client/swagger/unity.yml
Comment (Asalem1) [reply src/client/swagger/unity.yml:62]: Did this ever get changed to be a part of Billing rather than it's own definition?

File: src/client/swagger/unity.yml
Comment (Asalem1) [inline src/client/swagger/unity.yml:180]: Quick question: why is the billing contact a separate API query here? Wouldn't that be considered part of the [billing information](https://github.com/influxdata/ui/pull/666/files#diff-888a46184afed8040a74345087d09db51553252713b486dbd57610e728c89d02R33)?

File: src/client/swagger/unity.yml
Comment (Asalem1) [inline src/client/swagger/unity.yml:1003]: Are the Limits necessary? The UI currently gets that data from IDPE

File: src/client/swagger/unity.yml
Comment (Asalem1) [inline src/client/swagger/unity.yml:459]: I think this is already implemented here:

https://github.com/influxdata/ui/blob/master/src/cloud/components/GetAssetLimits.tsx

Where we get this https://github.com/influxdata/ui/blob/master/src/cloud/apis/limits.ts#L20

Which is defined here:

https://github.com/influxdata/ui/blob/master/src/types/cloud.ts#L1

File: src/client/swagger/unity.yml
Comment (randycoulman) [reply src/client/swagger/unity.yml:180]: Billing contact is a separate entity in Quartz. It is added on the checkout page, but can be viewed and edited on the billing page.

File: src/client/swagger/unity.yml
Comment (randycoulman) [reply src/client/swagger/unity.yml:459]: Assuming that the user has permissions to read those limits, it's probably fine to get them directly from IDPE. I thought we needed Quartz' operator token for that, but if not, then we're good there.

File: src/client/swagger/unity.yml
Comment (randycoulman) [reply src/client/swagger/unity.yml:1003]: We'll need them for the operator UI because it provides the ability to change limits for a user. A user wouldn't have permission to change their own limits.

File: src/client/swagger/unity.yml
Comment (Asalem1) [reply src/client/swagger/unity.yml:180]: Gotcha. Thanks!

File: src/client/swagger/unity.yml
Comment (Asalem1) [reply src/client/swagger/unity.yml:459]: Ya, I don't think there should be an issue getting it from IDPE. Especially since this implementation won't focus on integrating Cancelled users

File: src/client/swagger/unity.yml
Comment (Asalem1) [reply src/client/swagger/unity.yml:1003]: Gotcha.  I think i bundled this up with the org limits stuff in the other comment. That makes sense, not entirely certain of the dynamic but it seems unique

File: src/client/swagger/unity.yml
Comment (randycoulman) [reply src/client/swagger/unity.yml:62]: No, it didn't. Feel free to move it where it makes sense for you, or we can look at options when we implement the usage page API.

Comment (Asalem1) [reply src/client/swagger/unity.yml:62]: Great! Thanks Randy!

File: src/client/swagger/unity.yml
Comment (ischolten) [reply src/client/swagger/unity.yml:459]: So this limits section will need to exist in the quartz api for the operator pages (not for the billing ones though). I think we can lock it down to be an operator. The main reason for this is  if I am an operator, i need to be able to view and update limits for any org no matter if my idpe user has access to it or not. For this reason, we will need to get these through quartz because the quartz token does have access to view these.

File: src/client/swagger/unity.yml
Comment (ischolten) [reply src/client/swagger/unity.yml:180]: This makes me think that there also needs to be a get for /billing/contact for displaying the existing billing contact on the billingn page. Rather than just updating it.

Comment (ischolten) [reply src/client/swagger/unity.yml]: Oh just kidding, i see thats a part of just `/billing`. That seems weird to me, but ill leave it for now.

File: src/client/swagger/unity.yml
Comment (ischolten) [reply src/client/swagger/unity.yml:62]: Im going to leave this as is for now unless, @Asalem1  wants to rename it or move it. I dont want to keep adding things to `/billing` since that endpoint seems to be providing more of a blob of things to render on the page rather than an actual resource.

**Comment by hercules-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 43 | :red_circle:
accessibility | 96 | :green_circle:
best-practices | 93 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :question:

File: src/billing/components/BillingPage.tsx
Comment (Asalem1) [inline src/billing/components/BillingPage.tsx:56]: removed isCancelled since it's no longer relevant to the UI (i.e. Cancelled users can't access the app)

File: src/billing/components/BillingPageContents.tsx
Comment (Asalem1) [inline src/billing/components/BillingPageContents.tsx:20]: Removed cancelled since it's no longer relevant to the UI

File: src/usage/UsagePage.tsx
Comment (Asalem1) [inline src/usage/UsagePage.tsx:24]: Still need to see if this is relevant for the Operator page

Comment (ischolten) [reply src/usage/UsagePage.tsx:24]: I believe the AlertStatusCancelled was used for the old operator ui and is only used on regular user pages like billing, users, and usage. So it should be fine to delete.

Comment (Asalem1) [reply src/usage/UsagePage.tsx]: Cool, removed it, thanks!

**Pull request merged**
